src/less/filter-button/filter-button.less

```diff
@@ -38,19 +38,14 @@ a.filter-link:visited {
 
 button.filter-button[aria-pressed="true"],
 a.filter-link--selected {
-    .background-color-token(filter-button-selected-background-color, color-background-inverse);
-    .color-token(filter-button-selected-foreground-color, color-foreground-on-inverse);
+    border: 1px solid;
```

Comment:
You should include the border color via `.border-color-token` here. That should make the look and feel more "bulletproof," less reliant on inheritance, and allowed to be more easily modified for dark mode.